flocker/node/test/istatechange.py

```diff
@@ -17,9 +17,9 @@
 from pyrsistent import PClass, field
 from characteristic import attributes
 
-from twisted.trial.unittest import SynchronousTestCase
 from twisted.internet.defer import succeed
 
+from flocker.testtools import TestCase
```

Comment:
Changed, but for the record, I think relative imports of testtools modules are more confusing.